header.md

```diff
@@ -6,7 +6,7 @@ If you have any problems, please create an issue on [GitHub](https://github.com/
 
 ### API vs S3 Data Retention
 
-Currently the API only supports a rolling 90 day archive of the data. For access to the full dataset, you can access S3 directly by going [here](https://openaq-fetches.s3.amazonaws.com/index.html) or [here](https://openaq-data.s3.amazonaws.com/index.html). Read [this explainer](https://medium.com/@openaq/how-in-the-world-do-you-access-air-quality-data-older-than-90-days-on-the-openaq-platform-8562df519ecd) on how to use AWS Athena for historical data access.
+Currently the API supports a 2 year archive of the data. For access to the full dataset, you can access S3 directly by going [here](https://openaq-fetches.s3.amazonaws.com/index.html) or [here](https://openaq-data.s3.amazonaws.com/index.html). Read [this explainer](https://medium.com/@openaq/how-in-the-world-do-you-access-air-quality-data-older-than-90-days-on-the-openaq-platform-8562df519ecd) on how to use AWS Athena for historical data access.
```

Comment:
Here links are not recommended https://webaccess.berkeley.edu/ask-pecan/click-here

Maybe from [OpenAQ fetches thing] or [OpenAQ data thing] where "thing" is replaced with a smart word?